action.yml

```diff
@@ -53,9 +53,21 @@ runs:
 
   # Checks that all Terraform configuration files adhere to a canonical format
     - name: Terraform Format
-      run: terraform fmt -check
+      id: check_formating
+      run: terraform fmt -check  
+      continue-on-error: true  
       shell: bash
 
+    - name: Terraform Format Status
```

Comment:
both of these have the same logic?

Reply:
Yes, I tried to put the exit 1 from the next step after the echo in the same step but it didn't work. So split in into 2 separate steps and it now works. Is there a better way of doing this?